src/pages/kb/user-guide/dashboards/dashboard-editing.md

```diff
@@ -63,3 +63,18 @@ Now the Dashboard options menu includes a `Manage Permissions` option. Clicking
 
 Please note that currently the users you add won't receive a notification, so you will need to notify them manually.
 
+## Dashboard Refresh
+
+When you open a dashboard, Redash displays the most recent cached query result for each visualization. Dashboard data may grow stale as a result, depending on how often you run the underlying queries (either manually or with a scheduled query). However, you can force Redash to execute all of the underlying queries by clicking the Refresh button on the upper-right. 
```

Comment:
Maybe link `scheduled query` to the relevant section?

Reply:
@ranbena handled this.